java/gui/src/main/java/com/squareup/subzero/SubzeroGui.java

```diff
@@ -257,7 +257,30 @@ private void signTxTest() throws Exception {
         byte[] proto = base64().decode(request);
 
         CommandRequest commandRequest = CommandRequest.parseFrom(proto);
+      if (filePathInJAR.contains("negative")) {
+          try {
+              CommandResponse response =
+                      CommandHandler.dispatch(this, new InternalCommandConnector(hostname, port),
+                              commandRequest);
+          } catch (Exception e) {
+              // A list of negative test cases can be added here.
+              if (
+                      filePathInJAR.contains("bad_qrsignature") && e.toString().contains("SIG_CHECK_FAILED") ||
+                              (true)
```

Comment:
logic here is a bit complicated to follow. what's the `|| true` used for?

Reply:
thanks for pointing that out. That was a total mistake. My bad. It should not be here.